### PR TITLE
swift-format 5.1 (new formula)

### PR DESCRIPTION
**Pull request by goranmoomin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/swift-format.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class SwiftFormat < Formula
+  desc "Formatting technology for Swift source code"
+  homepage "https://github.com/apple/swift-format"
+  url "https://github.com/apple/swift-format.git", :revision => "fc5a3da1d5d03143d7a31b12514349e2bf1aba8f"
```
Comment (SMillerDev) [inline Formula/swift-format.rb]: This needs to point at a tagged release.

File: Formula/swift-format.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  version "swift-5.1"
```
Comment (SMillerDev) [inline Formula/swift-format.rb]: Can't the version just be 5.1 then?

File: Formula/swift-format.rb (hunk 1 above)
Comment (pcr910303) [reply Formula/swift-format.rb]: Hmm, this repo doesn't have a tagged release - it's 'release' is the (frozen) branch 'swift-5.1-branch'. Would that mean this formula is not acceptable for Homebrew?

File: Formula/swift-format.rb (hunk 2 above)
Comment (pcr910303) [reply Formula/swift-format.rb]: Won't matter, I think.

Comment (SMillerDev) [reply Formula/swift-format.rb]: It will for homebrew parsing.

File: Formula/swift-format.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/swift-format.rb]: No: https://docs.brew.sh/Acceptable-Formulae#stable-versions

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @pcr910303 for your contribution to Homebrew! :tada: 🥇

**Pull request closed without merging**